rct229/ruletest_engine/ruletest_jsons/ashrae9012022/section6/rule_6_10.json

```diff
@@ -0,0 +1,292 @@
+                                                            {
+                                                                "id": "Lighting 1",
+                                                                "purpose_type": "RETAIL_DISPLAY",
+                                                                "power_per_area": 6.99654177086132
+                                                            }
+                                                        ]
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "type": "PROPOSED"
+                    }
+                ]
+            },
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_descriptions": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "lighting_space_type": "RETAIL_FACILITIES_MALL_CONCOURSE",
+                                                        "floor_area": 92.90303999999999,
+                                                        "interior_lighting": [
+                                                            {
+                                                                "id": "Lighting 1",
+                                                                "purpose_type": "RETAIL_DISPLAY",
+                                                                "power_per_area": 6.99654177086132
+                                                            }
+                                                        ]
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "type": "BASELINE_0"
+                    }
+                ]
+            }
+        }
+    },
+    "rule-6-10-c": {
```

Comment:
1. Should this case be (b)? 
2. This case also falls into undetermined because ```baseline_interior_display_w``` and ```proposed_interior_display_w``` aren't the same and ```proposed_interior_display_w``` > ```minimum_retail_display_w```,

Reply:
Hmm the test appears correct however. Here is the RDS I'm following:
https://github.com/pnnl/ruleset-checking-tool/blob/d418aaf5fb1195a96eeb2bd12fdf18207b78857b/docs/section6/Rule6-10.md

`elif baseline_interior_display_w > min(proposed_interior_display_w, maximum_retail_display_w): FAIL` 

Baseline interior display is 1.4 W/ft2, proposed interior is 1.2, and maximum retail space is 1.75 W/ft2 ((1000+750)/1000).

Therefore

`1.4 > min(1.2, 1.75)`
`1.4 > 1.2` 

Am I missing something?

Reply:
Thank you for checking Juan! I believe in our code, we firstly check whether a case falls into undetermined in the ```manual_check_required``` function, and then pass/fail in the ```rule_check```. 

To make the case not undetermined first we should meet this logic (NOT (Case 1 and Case 2)) so, we should have values that meet ```elif baseline_interior_display_w **<** min(proposed_interior_display_w, maximum_retail_display_w)```.

Does this make sense to you?